t/t-batch-transfer.sh

```diff
@@ -247,3 +269,90 @@ begin_test "batch transfers with ssh endpoint (git-lfs-transfer)"
   git lfs fsck
 )
 end_test
+
+begin_test "batch transfers with ssh endpoint and multiple objects (git-lfs-transfer)"
+(
+  set -e
+
+  setup_pure_ssh
+
+  reponame="batch-ssh-transfer-multiple"
+  setup_remote_repo "$reponame"
+  clone_repo "$reponame" "$reponame"
+
+  contents1="test1"
+  contents2="test2"
+  contents3="test3"
+  git lfs track "*.dat"
+  printf "%s" "$contents1" >test1.dat
+  printf "%s" "$contents2" >test2.dat
+  printf "%s" "$contents3" >test3.dat
+  git add .gitattributes test*.dat
+  git commit -m "initial commit"
+
+  sshurl=$(ssh_remote "$reponame")
+  git config lfs.url "$sshurl"
+
+  # On Windows we do not multiplex SSH connections by default, so we
+  # enforce their use in order to match other platforms' connection counts.
+  git config --global lfs.ssh.autoMultiplex true
+
+  GIT_TRACE=1 git push origin main >push.log 2>&1
+  assert_ssh_transfer_sessions 'push.log' 'upload' 3 8
+  assert_remote_object "$reponame" "$(calc_oid "$contents1")" "${#contents1}"
+  assert_remote_object "$reponame" "$(calc_oid "$contents2")" "${#contents2}"
+  assert_remote_object "$reponame" "$(calc_oid "$contents3")" "${#contents3}"
+
+  cd ..
+  GIT_TRACE=1 git clone "$sshurl" "$reponame-2" 2>&1 | tee trace.log
```

Comment:
super minor nit: maybe call it `clone.log` for consistency?

Reply:
That's a good idea!  I've pushed a commit which renames those log files.